cmd/osbuild-worker/main.go

```diff
@@ -218,13 +223,23 @@ func main() {
 		}
 	}
 
+	var awsCredentials *credentials.Credentials
+	if config.AWS != nil {
+		awsCredentials = credentials.NewStaticCredentials(*config.AWS.AccessKeyId, *config.AWS.SecretAccessKey, "")
```

Comment:
I would use `NewSharedCredentials()`, and pass it the path to the file with the creds.

Reply:
In the end, I removed the initialization of the credentials here. It was not making a lot of sense.